week3/homework/Program.cs

```diff
@@ -0,0 +1,70 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+/* Create Account class that can be initialized with the amount value.
+ Account class contains Withdraw and Deposit methods and Balance (get only) property. 
+Make sure that you can't withdraw more than you have in the balance.*/
+
+public class Account{
+    private decimal balance;
+
+    public Account (decimal Balance)
```

Comment:
fields, method parameters should be camel cased. In this case it should be `balance`.